### DISCUSSION TITLE
No zoom on mobile webpage

### DISCUSSION THREAD

**Discussion by MasterTim17** [Hypfer/Valetudo#1641] [Q&A / Support] [answered]
Hello
I'm using version 2022.03.1 and recently can't zoom on the mobile webpage on my smartphone. I can zoom and start zoned cleaning when in desktop mode or with another smartphone. I already deleted all browser cache/data
What can I do?

**Comment by Hypfer** [comment, marked as answer]
There seems to be an issue with recent Blink versions, where `DOMMatrix.invertSelf()` occasionally turns a `1` into a `0.999999999999999` leading to all 2d transformations to fail as the browser now thinks that it is a 3d matrix.

Unfortunately, it seems that this regression will hit all users of chrome, edge etc in the upcoming weeks when they auto-update their browsers.

Valetudo 2022.11 contains a workaround against that.
If you decide to update, make sure to check out all the release notes as quite a few things have changed
